ArgusCore/src/main/java/com/salesforce/dva/argus/service/alert/DefaultAlertService.java

```diff
@@ -290,6 +290,10 @@ public Alert findAlertByPrimaryKey(BigInteger id) {
 		List<History> historyList = new ArrayList<>();
 		List<AlertIdWithTimestamp> alertIdWithTimestampList = _mqService.dequeue(ALERT.getQueueName(), AlertIdWithTimestamp.class, timeout,
 				alertCount);
+		if(alertIdWithTimestampList.isEmpty()) {
+			_logger.info("no alerts due");
```

Comment:
revert this, I made a separate commit (https://github.com/salesforce/Argus/pull/344/files) that has a PR against the public repo and is already merged to kiwigrid-develop